java/sage/ZRoot.java

```diff
@@ -35,19 +35,16 @@ public ZRoot(UIManager inUIMgr)
     super();
     uiMgr = inUIMgr;
     vf = uiMgr.getVideoFrame();
-    if (!Sage.EMBEDDED)
-    {
-      addComponentListener(this);
-      addMouseListener(this);
-      addMouseMotionListener(this);
-      addHierarchyListener(this);
-      vf.addMouseListeners(this);
-      vf.addMouseMotionListeners(this);
-    }
+    addComponentListener(this);
+    addMouseListener(this);
+    addMouseMotionListener(this);
+    addHierarchyListener(this);
+    vf.addMouseListeners(this);
+    vf.addMouseMotionListeners(this);
 
     alive = true;
 
-    if (Sage.EMBEDDED || uiMgr.getBoolean("ui/remote_render", false))
+    if (uiMgr.getBoolean("ui/remote_render", false))
```

Comment:
This might be a red herring.... I was running my local server with HEADLESS=false so that I could do some studio work, and this error is coming from that.  If I set my server to HEADLESS=true then the error goes away.